src/resmom_win/start_exec.c

```diff
@@ -2258,10 +2231,18 @@ finish_exec(job *pjob)
 			}
 			si.hStdInput = hReadPipe_dummy;
 		}
-		if (script_out != -1)
+		if (script_out != -1) {
 			si.hStdOutput = (HANDLE)_get_osfhandle(script_out);
-		if (script_err != -1)
+			if (si.hStdOutput == -1) {
+				log_err(-1, __func__, "Unable to get handle for the script output file");
+			}
+		}
+		if (script_err != -1) {
 			si.hStdError = (HANDLE)_get_osfhandle(script_err);
+			if (si.hStdError == -1) {
+				log_err(-1, __func__, "Unable to get handle for the script error file");
+			}
+		}
 		/* turn off echoing if cmd.exe */
 		if (strstr(shell, "cmd") ||
```

Comment:
Please add for which cmdline createProcess failed 
https://github.com/PBSPro/pbspro/blob/06f24712f1b101a386d5b88656a24489250e52f0/src/resmom_win/start_exec.c#L2644